### PR TITLE
my plugin description have been updated, support for mobile.  Please accept my pull request, thanks

### PR DESCRIPTION
**Pull request by zgj233**
my plugin had been updated, so I  want to update it description

### CODE REVIEW

**Comment by zgj233** [comment]
May I ask when my branches can be merged

**Comment by Jogai** [comment]
I'd suggest "A popup menu based on vue2.6+ including support for mobile" or "A popup menu for vuejs 2.6+ with support for mobile"

**Comment by zgj233** [comment]
OK，I have take  your advice .Thank you.
 so, my branches can be merged?

**Comment by zgj233** [comment]
Hello? 
May I ask when my branches can be merged?

**Comment by zgj233** [comment]
My plugin description have been updated, support for mobile. 
So, I want to update the describe for this plugin.
Please accept my pull request, thanks.

File: README.md [hunk 1]
```diff
@@ -1814,7 +1814,7 @@ Tooltips / popovers
 
  - [vue-context-menu](https://github.com/vmaimone/vue-context-menu) - A context menu component for vue js.
  - [vue-lil-context-menu](https://github.com/timwis/vue-lil-context-menu) - A flexible lil context menu component for Vue.
- - [vue-mouse-menu](https://github.com/zgj233/vue-mouse-menu) - A mouse-menu component for vue 2+.
+ - [vue-mouse-menu](https://github.com/zgj233/vue-mouse-menu) - A popup menu for vue 2.6+ with support for mobile.
```
Comment (Scrum) [inline README.md]: @zgj233 Needs to be at the end of the list.

**Pull request merged**